src/main/webapp/app/exam/participate/exam-navigation-bar/exam-navigation-bar.component.ts

```diff
@@ -49,14 +50,22 @@ export class ExamNavigationBarComponent implements OnInit {
         this.examAboutToEnd.emit();
     }
 
+    openExerciseOverview() {
+        // set index and emit event
+        this.exerciseIndex = 0;
+        // save current exercise
+        this.onPageChanged.emit({ overViewChange: true, exercise: this.exercises[this.exerciseIndex], forceSave: false });
+        this.setExerciseButtonStatus(this.exerciseIndex);
+    }
+
```

Comment:
Why are you not calling changeExercise here, they are almost the same?

Reply:
done